docs/cookbook/disabling-localised-urls.rst

```diff
@@ -0,0 +1,52 @@
+How to disable localised URLs?
+==============================
+
+**1.** Customise application routing in ``app/config/routing.yml``.
+
+Replace:
+
+.. code-block:: yaml
+
```

Comment:
Well, here there is no namespace :) The general convention for yaml files in the docs is to include a comment with filename.